public/app/features/live/pages/CloudAdminPage.tsx

```diff
@@ -15,9 +15,9 @@ export default function CloudAdminPage() {
 
   useEffect(() => {
     getBackendSrv()
-      .get(`api/live/remote-write-backends`)
+      .get(`api/live/write-configs`)
       .then((data) => {
-        setCloud(data.remoteWriteBackends);
+        setCloud(data.writeConfigs);
```

Comment:
Here is where I am not sure things will continue to work.

Reply:
Tested - works!

Reply:
:tada: easy when the front end does not do anything interesting